examples/bickleyjet/bickleyjet_cg_invariant_hypervisc.jl

```diff
@@ -110,7 +109,10 @@ function energy(state, p, local_geometry)
 end
 
 function rhs!(dydt, y, _, t)
-    D₄ = parameters.D₄
+    space = axes(y)
+    c = sqrt(parameters.g * parameters.ρ₀)
+    D₄ = 0.0015 * c * Spaces.lengthscale(space)^3 # hyperdiffusion coefficient
```

Comment:
@paullric Is this an appropriate way to choose the hyperdiffusion coefficient for the shallow water equations?